example/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <string name="app_name">Klarna On Demand Example</string>
+    <string name="app_name">Klarna Example App</string>
+    <string name="qrCodeImageDescription">qr code image</string>
```

Comment:
It's for accessibility - TTS(Text to speech) upon images to let people know what is on screen.